cdap-proto/src/main/java/co/cask/cdap/proto/metadata/lineage/FieldLineageDetails.java

```diff
@@ -24,29 +26,30 @@
 
 /**
  * Represents the lineage associated with the field of a dataset in detail.
- * Detail view in the backward direction not only includes the information about the fields
+ * Detail view in the incoming direction not only consists of the information about the fields
  * and corresponding datasets which were responsible for generating the field but also includes
  * the information about operations and programs which were responsible for generating the field.
- * Similarly detail view in the forward direction not only includes the target fields along with
- * their dataset information which got generated by this field but also includes the operations
- * and programs information which were responsible for the transformation.
+ * Similarly detail view in the outgoing direction not only consists of the target fields along with
```

Comment:
so you call this "target fields" or "destination fields"? Please be consistent (see DatasetField)

Reply:
The language can be simplified: 

Represents the lineage associated with the field of a dataset in detail. 
<ul>
<li>
In incoming direction, it consists of the datasets and their fields ({@link DatasetField}) that this field originates from, as well as the programs and operations that generated this field from those origins. 
</li><li>
In outgoing direction, it consists of the datasets and their fields ({@link DatasetField}) that were computed from this field, along with the programs and operations that performed the computation. 
</li>
</ul>

Reply:
fixed.